<launch>
  <arg name="input_cloud" default="/aeroa_remote/point_cloud/cloud_registered/remote" />
  <arg name="input_image" default="/aeroa_remote/openni_camera/rgb/image_rect_color" />
  <arg name="input_cluster" default="/multi_euclidean_clustering/output_indices" />
  <arg name="prefix" default="$(env HOME)/.ros/neatness_estimator" />

  <arg name="fe_service_topic" default="/objects_feature_extractor/extract" />
  <arg name="de_service_topic" default="/distance_estimator/estimate" />

  <node name="estimation_module_interface_color_and_geometry"
        pkg="nodelet" type="nodelet"
        args="standalone neatness_estimator/EstimationModuleInterface"
        output="screen">
    <remap from="~input_cloud" to="$(arg input_cloud)" />
    <remap from="~input_image" to="$(arg input_image)" />
    <remap from="~input_cluster" to="$(arg input_cluster)" />

    <rosparam subst_value="true">
      approximate_sync: true
      prefix: $(arg prefix)
      fe_service_topic: $(arg fe_service_topic)
      de_service_topic: $(arg de_service_topic)
      get_color_mask: true
      only_color_and_geometry: true
    </rosparam>

  </node>

</launch>
